### PR TITLE
Add support page, minor edits to other pages

### PR DESCRIPTION
**Pull request by rpellows**

### CODE REVIEW

**Review by mrmccarthy — APPROVED** [review]
LGTM

**Review by vjtirumalai — APPROVED** [review]
LGTM

File: docs/community.md [hunk 1]
```diff
@@ -55,8 +55,7 @@ improve existing processes and create new ones.
 
 **Learn More**
 
-* [Managing project and tool permissions and memberships](maintainers_corner.md)
-* [Legal processes](legal.md)
```
Comment (shpandrak) [inline docs/community.md:59]: Did you mean to remove the "Legal process" link from here?

Comment (rpellows) [reply docs/community.md:59]: Yes.  The legal requirements are captured in the maintainers' page.  We may do away with legal.md or make it point to the maintainers' page.

**Pull request merged**